### PR TITLE
[ADD] iot_option_oca

### PR DESCRIPTION
**Pull request by etobella**
New module for managing custom options on devices

@lubusax

TODO:

- [x] readme
- [x] Improve tests

### CODE REVIEW

File: iot_option_oca/models/ir_actions_act_window_view.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class IrActionsActWindowView(models.Model):
+    _inherit = "ir.actions.act_window.view"
+
+    view_mode = fields.Selection(selection_add=[("iot_option", "Iot Option")])
```
Comment (pedrobaeza) [inline iot_option_oca/models/ir_actions_act_window_view.py]:
```suggestion
    view_mode = fields.Selection(selection_add=[("iot_option", "IoT Option")])
```

File: iot_option_oca/models/ir_ui_view.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class IrUiView(models.Model):
+    _inherit = "ir.ui.view"
+
+    type = fields.Selection(selection_add=[("iot_option", "Iot Option")])
```
Comment (pedrobaeza) [inline iot_option_oca/models/ir_ui_view.py]:
```suggestion
    type = fields.Selection(selection_add=[("iot_option", "IoT Option")])
```

**Review by pedrobaeza — COMMENTED** [review]
Can you please run pre-commit for having the setup folder for trying this on runboat?

**Review by lubusax — APPROVED** [review]

**Pull request merged**